test/allowlist/gcc/glibc.log

```diff
@@ -1,25 +1,67 @@
 #
 # XXX: Need review why.
 #
-FAIL: gfortran.dg/matmul_15.f90
-FAIL: gfortran.dg/ieee/ieee_1.F90
-FAIL: gfortran.dg/ieee/ieee_2.f90
 FAIL: gfortran.dg/ieee/ieee_6.f90
-FAIL: gfortran.dg/ieee/large_2.f90
-FAIL: gfortran.dg/ieee/large_3.F90
 FAIL: gfortran.dg/ieee/modes_1.f90
-FAIL: gfortran.dg/ieee/rounding_1.f90
 #
 # Fails if there is no F extension
 #
 FAIL: gfortran.dg/ieee/rounding_3.f90
 #
-# Synchronization problem.
+# Fixed but not backport
 #
-FAIL: gcc.dg/tree-prof/time-profiler-2.c
+UNRESOLVED: gcc.target/riscv/rvv/autovec/vls-vlmax/full-vec-move1.c
+FAIL: gcc.target/riscv/rvv/autovec/vls-vlmax/full-vec-move1.c
```

Comment:
Ooops, could you point me which commit fix that? so that I could backport that and we will have fix on 14.3...

Reply:
I think it is https://gcc.gnu.org/git/?p=gcc.git;a=commit;h=b1520d2260c5e0cfcd7a4354fab70f66e2912ff2,
I compare the tree form trunk with the gcc14.2 release tree https://gcc.gnu.org/git/?p=gcc.git&a=search&h=04696df09633baf97cdbbdd6e9929b9d472161d3&st=commit&s=full-vec-move1.c